crmsh/utils.py

```diff
@@ -2287,4 +2253,58 @@ def interface_choice():
     # should consider interface format like "ethx@xxx"
     interface_list = re.findall(r'(?:[0-9]+:) (.*?)(?=: |@.*?: )', out)
     return [nic for nic in interface_list if nic != "lo"]
+
+
+def interface_info(ipv6=False):
+    """
+    Get network information(IPv4 default) in local, include:
+        nic-ip dictionary
+        ip list
+        ip with mask list
+    IPv6 support
+    """
+    nic_ip_dict = {}
+    ip_list = []
+    ip_with_mask_list = []
+
+    cmd = "ip -{} -o addr show".format(6 if ipv6 else 4)
+    rc, out, err = get_stdout_stderr(cmd)
+    if rc != 0:
+        raise ValueError(err)
+
+    for line in out.split('\n'):
```

Comment:
You could just use `splitlines()`

Reply:
Well, `splitlines()` looks good, it can strip last '\n', while `split('\n')` will not